nodemon.json

```diff
@@ -1,6 +1,5 @@
 {
-  "watch": ["src"],
-  "ext": "ts",
-  "ignore": ["src/**/*.spec.ts"],
-  "exec": "ts-node -r tsconfig-paths/register src/main.ts"
+  "watch": ["dist"],
+  "ext": "js",
+  "exec": "node -r tsconfig-paths/register -r ts-node/register dist/main.js"
```

Comment:
```
 -r tsconfig-paths/register -r ts-node/register 
```
Why are these required?

Reply:
> ```
>  -r tsconfig-paths/register -r ts-node/register 
> ```
> Why are these required?

`tsconfig-paths` is used when people use `paths` in their `tsconfig.json` to map things in.

`ts-node/register` can be dropped

Reply:
> tsconfig-paths is used when people use paths in their tsconfig.json to map things in.

Are you sure that it will still work after compilation? `tsconfig-paths` is used with combination with the `ts-node`